week3/src/components/Content.jsx

```diff
@@ -0,0 +1,179 @@
+import styled from "styled-components";
+import React, { useState } from "react";
+import { quizList } from "./QuizInfo";
+import Modal from "./Modal";
+
+quizList.sort(() => Math.random() - 0.5);
+let i = 0;
+
+function Content() {
+  //답
+  const [answer, setAnswer] = useState(quizList[i]);
+
+  //단계
+  const [stage, setStage] = useState(1);
+
+  //끝났음,
+  const [isOver, setIsOver] = useState(false);
+  const [isCorrect, setIsCorrect] = useState(false);
+
+  //모달
+  const [isShown, setIsShown] = useState(false);
+
+  function checkAnswer(chosen) {
+    //맞았을 떄
+    if (chosen === answer.ans) {
+      i++;
+      setIsCorrect(true);
+
+      setAnswer(quizList[i]);
+      answer.option.sort(() => Math.random() - 0.5);
+
+      if (stage !== quizList.length) {
+        setStage(stage + 1);
+      } else {
+        setIsOver(true);
+      }
```

Comment:
```suggestion
      stage !== quizList.length ? setStage(stage + 1) : setIsOver(true);
```
이런 경우 삼항연산자로 코드를 간단하게 줄일 수 있을 것 같아용!

Reply:
오! 삼항 연산자 쓰는 게 아직은 익숙하지 않아서 계속 if~else로 짜게 되네요ㅜㅜ 반영해서 고쳐볼게욥! 감삼당!💚